pallets/phala/src/mock.rs

```diff
@@ -130,14 +131,17 @@ impl mining::Config for Test {
 	type OnReclaim = PhalaStakePool;
 	type OnStopped = PhalaStakePool;
 	type OnTreasurySettled = ();
+	type UpdateTokenomicOrigin = frame_system::EnsureRoot<u64>;
```

Comment:
`frame_system::EnsureRoot<Self::AccountId>` is better

Reply:
Done